views/default/elements/navigation.css.php

```diff
@@ -436,38 +436,49 @@
 	position: absolute;
 	z-index: 10000;
 	overflow: hidden;
-	min-width: 180px;
-	max-width: 250px;
 	border: 1px solid #DEDEDE;
 	background-color: #FFF;
+	margin-right: 10px;
 
 	border-radius: 0 3px 3px 3px;
 	box-shadow: 1px 3px 5px rgba(0, 0, 0, 0.25);
 }
-.elgg-menu-hover > li {
-	border-bottom: 1px solid #ddd;
-}
-.elgg-menu-hover > li:last-child {
-	border-bottom: none;
+
+.elgg-menu-hover-card-container {
+	display: flex;
```

Comment:
csscrush will handle this, but I agree this PR should stand alone as the codebase is today.